pallas-applying/tests/byron.rs

```diff
@@ -11,77 +10,99 @@ use pallas_codec::{
         decode::{Decode, Decoder},
         encode,
     },
-    utils::{CborWrap, EmptyMap, KeepRaw, MaybeIndefArray, TagWrap},
-};
-use pallas_crypto::hash::Hash;
-use pallas_primitives::byron::{
-    Address, Attributes, MintedTxPayload as ByronTxPayload, Tx as ByronTx, TxId as ByronTxId,
-    TxIn as ByronTxIn, TxOut as ByronTxOut, Witnesses as ByronWitnesses,
+    utils::{CborWrap, KeepRaw, MaybeIndefArray, TagWrap},
 };
+use pallas_primitives::byron::{Address, MintedTxPayload, Twit, Tx, TxIn, TxOut, Witnesses};
 use pallas_traverse::{MultiEraInput, MultiEraOutput, MultiEraTx};
 
 #[cfg(test)]
 mod byron_tests {
     use super::*;
 
-    #[test]
-    // Note that:
-    //      i)   the transaction input contains 100000 lovelace,
-    //      ii)  the minimum_fee_constant protocol parameter is 7,
-    //      iii) the minimum_fee_factor protocol parameter is 11, and
-    //      iv)  the size of the transaction is 82 bytes—it is easy to verify
-    //              that 82 == pallas_applying::get_byron_tx_size(tx).
-    // The expected fees are therefore 7 + 11 * 82 = 909 lovelace, which is why
-    // the output contains 100000 - 909 = 99091 lovelace.
-    fn successful_case() {
-        let protocol_params: ByronProtParams = ByronProtParams;
-        let mut tx_ins: ByronTxIns = empty_tx_ins();
-        let tx_in: ByronTxIn = new_tx_in(rand_tx_id(), 3);
-        add_byron_tx_in(&mut tx_ins, &tx_in);
-        let mut tx_outs: ByronTxOuts = new_tx_outs();
-        let tx_out_addr: Address = new_addr(rand_addr_payload(), 0);
-        let tx_out: ByronTxOut = new_tx_out(tx_out_addr, 99091);
-        add_tx_out(&mut tx_outs, &tx_out);
+    const MAINNET_TX_HEX_CBOR: &str =
+        "82839f8200d8185824825820da832fb5ef57df5b91817e9a7448d26e92552afb34f8ee5adb491b24bbe990d50e\
+        ff9f8282d818584283581cdac5d9464c2140aeb0e3b6d69f0657e61f51e0c259fe19681ed268e8a101581e581c2\
+        b5a44277e3543c08eae5d9d9d1146f43ba009fea6e285334f2549be001ae69c4d201b0000000172a84e408282d8\
+        18584283581c2b8e5e0cb6495ec275872d1340b0581613b04a49a3c6f2f760ecaf95a101581e581cca3e553c9c6\
+        3c5b66689e943ce7dad7d560ae84d7c2eaf21611c024c001ad27c159a1b00000003355d95efffa0818200d81858\
+        85825840888cdf85991d85f2023423ba4c80d41570ebf1fc878c9f5731df1d20c64aecf3e8aa2bbafc9beba8ef3\
+        3acb4d7e199b445229085718fba83b7f86ab6a3bcf782584063e34cf5fa6d8c0288630437fa5e151d93907e826e\
+        66ba273145e3ee712930b6f446ff81cb91d7f0cb4ceccd0466ba9ab14448d7eab9fc480a122324bd80170e";
```

Comment:
Not required for now, but we should load test data from the `test_data` folder where we already hold test vectors for different crates.

Reply:
In [this commit](https://github.com/txpipe/pallas/commit/aeb1a1a268a901cae201570637e831dec01a1b42) I moved the CBOR to test_data